cognite_toolkit/cdf_tk/templates/_utils.py

```diff
@@ -45,3 +50,31 @@ def iterate_functions(module_dir: Path) -> Iterator[list[Path]]:
         function_directories = [path for path in function_dir.iterdir() if path.is_dir()]
         if function_directories:
             yield function_directories
+
+
+def _get_cognite_module_version(project_dir: Path) -> str:
+    cognite_modules = project_dir / "cognite_modules"
+    if (cognite_modules / "_system.yaml").exists():
+        system_yaml = read_yaml_file(cognite_modules / "_system.yaml")
+        try:
+            previous_version = system_yaml["cdf_toolkit_version"]
+        except KeyError:
+            previous_version = None
+    elif (project_dir / "environments.yaml").exists():
+        environments_yaml = read_yaml_file(project_dir / "environments.yaml")
+        try:
+            previous_version = environments_yaml["__system"]["cdf_toolkit_version"]
+        except KeyError:
+            previous_version = None
+    else:
+        previous_version = None
+    if previous_version is None:
+        print(
+            "Failed to load previous version, have you changed the "
+            "'_system.yaml' or 'environments.yaml' (before 0.1.0b6) file?"
+        )
+        exit(1)
+    if previous_version == current_version:
+        print("No changes to the toolkit detected.")
+        exit(0)
```

Comment:
Sort of strange to do exit() in a _function?

Reply:
Agreed, moved up.